dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateFactoryImpl.java

```diff
@@ -318,8 +316,48 @@ public void associateContainers(List<Container> containerIdentifiers,Template te
 		}
 		return result;
 	}
-	
-	private List<String> getContainerIds(String templateBody) {
+
+	private Collection<String> getContainerIds(Template template) {
+		try {
+			return this.getContainerIdsFromJSON(template);
+		} catch (Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)